docs/best_practices/nwbfile_metadata.rst

```diff
@@ -11,18 +11,26 @@ File Organization
 
 .. _best_practice_global_time_reference:
 
-Global Time Reference
-~~~~~~~~~~~~~~~~~~~~~
+Global Date and Time Reference
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-An :nwb-schema:ref:`sec-NWBFile` can have two primary time references. The global time reference for all objects in the
-:nwb-schema:ref:`sec-NWBFile` is the ``timestamps_reference_time``. By default, this is also set to the
-``session_start_time``, but when writing multiple NWBFiles that are all designed to align
-to the same time reference, the ``session_start_time`` may be set separately from the explicitly specified common
-``timestamps_reference_time`` used across all of the NWBFiles.
+An :nwb-schema:ref:`sec-NWBFile` can have two primary time references. The global date and time reference for all
+objects in the :nwb-schema:ref:`sec-NWBFile` is the ``timestamps_reference_time``. By default, this is also set to the
+``session_start_time``, but when writing multiple NWBFiles that are all designed to align to the same time reference,
+the ``session_start_time`` may be set separately from the explicitly specified common ``timestamps_reference_time``
+used across all of the NWBFiles.
 
 All time-related data in the NWBFile should be synchronized to the ``timestamps_reference_time`` so that future users
 are able to understand the timing of all events contained within the NWBFile.
 
+Given the importance of this field within an :nwb-schema:ref:`sec-NWBFile`, is it critical that it be set to a proper
+value. Default values should generally not be used for this field. If the true date and time are unknown, use your
+best guess or the date and time of writing the :nwb-schema:ref:`sec-NWBFile`.
```

Comment:
Realistically, it is common for labs to not know the time of an experiment. Maybe we can speak to this.

Reply:
Changed to

"If the true date is unknown, use your best guess. If the exact start time is unknown, then it is fine to simply set it to midnight on that date."

Reply:
We could I suppose, but it's not as if we are often faced with sessions where we know a time plus-or-minus 2 hours. It's generally the case that we either know the time to the second or to the day. I think it would be a lot more straightforward to just allow for the session_start_time to be a date and not a datetime

Reply:
@oruebel I would second what @bendichter proposes. I forget the docval language, but in annotation typing it would make sense for

```python
NWBFile(
    session_start_time: Union[datetime.date, datetime.datetime],
    ...
)
```

Many of the Buzsaki DANDISets have this issue actually; either one of their acquisition systems grabbed the exact start time down to the second, or it didn't but they still recorded the date of the experiment in the file name.

Reply:
Of course best practice would be datetime.